ethcore/src/state_db.rs

```diff
@@ -58,7 +58,7 @@ struct CacheQueueItem {
 	/// Account address.
 	address: Address,
 	/// Acccount data or `None` if account does not exist.
-	account: Option<Account>,
+	account: Mutex<Option<Account>>,
```

Comment:
Why is there a mutex required here? Not sure that creating thousands of mutexes is a good idea.